relayer/pathEnd.go

```diff
@@ -109,7 +109,7 @@ func (src *PathEnd) ChanInit(dst *PathEnd, ordering chanState.Order, signer sdk.
 	return chanTypes.NewMsgChannelOpenInit(
 		src.PortID,
 		src.ChannelID,
-		defaultIBCVersion,
+		defaultTransferVersion,
```

Comment:
Note, a future PR should refactor this so that relayer doesn't just support transfer channels. We can do this by making the channel version a part of PathEnd, or something similar